docs/installation/autoware/source-installation.md

```diff
@@ -2,7 +2,17 @@
 
 ## Prerequisites
 
-- [Ubuntu 20.04](https://releases.ubuntu.com/20.04/)
+- OS
+
+  - [Ubuntu 20.04](https://releases.ubuntu.com/20.04/) (**active development**)
+  - [Ubuntu 22.04](https://releases.ubuntu.com/22.04/) (**will be supported in 2022**)
+
+- ROS
+
+  - ROS 2 Galactic (**active development**)
+  - ROS 2 Humble (**will be supported in 2022**)  
+    For the system dependencies, refer to [REP-2000](https://www.ros.org/reps/rep-2000.html) .
```

Comment:
- I think it is better to remove `active development`. If the OS/ROS version is listed, then it is supported - we shouldn't need to qualify that support by saying it is being actively developed.
- IIRC we are planning to support 22.04 and Humble from about Q3, so I think it is better to give a better idea of when those versions will be supported rather than saying "2022", and then having to answer questions about when in 2022.
- Also, made a change to make it clear that the system dependencies are for ROS 2

```suggestion
  - [Ubuntu 20.04](https://releases.ubuntu.com/20.04/)
  - [Ubuntu 22.04](https://releases.ubuntu.com/22.04/) (**will be supported from Q3 2022**)

- ROS
  - ROS 2 Galactic
  - ROS 2 Humble (**will be supported from Q3 2022**)  
    For ROS 2 system dependencies, refer to [REP-2000](https://www.ros.org/reps/rep-2000.html) .
```